File: java-lhdiff-project/datasets/Aakanksha/File04_Old.java
// File04_Old.java
public class File04 {
    public void processConfiguration(String host, int port, String username, String password) {
        // A very long initialization line
        String connectionString = "jdbc:mysql://" + host + ":" + port + "/" + username + "?password=" + password + "&useSSL=false";
        System.out.println("Connecting to: " + connectionString);
        // More code...
    }
}

File: python-lhdiff-project/datasets/Aakanksha/File04_Old.py
# File04_Old.py

class File04:
    def process_configuration(self, host, port, username, password):
        # A very long initialization line
        connection_string = (
            f"jdbc:mysql://{host}:{port}/{username}?password={password}&useSSL=false"
        )
        print("Connecting to:", connection_string)
        # More code...